CHANGELOG.md

```diff
@@ -26,6 +26,16 @@ INTERNAL NOTE
 -->
 ## Flutter 3.24 Changes
 
```

Comment:
Were there any Dart changes included in this hotfix?

Reply:
https://github.com/dart-lang/sdk/blob/stable/CHANGELOG.md#352---2024-08-28

Reply:
Dart patch notes updated to the format Dart uses.